Dockerfile

```diff
@@ -4,8 +4,8 @@
 FROM nginx:alpine
 MAINTAINER Mainflux
 
-ENV MFX_CORE_HOST mainflux-core
-ENV MFX_CORE_PORT 7070
+ENV MFX_MANAGER_HOST mainflux-manager
+ENV MFX_MANAGER_PORT 7070
```

Comment:
This one moved to [9090](https://github.com/mainflux/mainflux-manager/blob/master/main.go#L64-L65). But OK, I can fix this quickly upon merge.